File: patterns/js/pattern-13.js
/**

1        1
12      21
123    321
1234  4321
1234554321

 */

let line= '';
let N=9;
let end;
for(let i=1;i<=N;i++){
    for(let j=1;j<=i;j++){
        line+=`${j}`;
        end=j
    }
    let spaces = 2*(N-i);
    if(spaces){
        for(let j=0;j<spaces;j++){
            line+=' '
        }
    }

    for(let j=end;j>=1;j--){
        line+=`${j}`;
        end=''
    }
    console.log(line);
    line='';
    end=''
}

File: patterns/python/pattern13.py
'''
1        1
12      21
123    321
1234  4321
1234554321
'''

n=5
for i in range(1,n+1):
    for j in range(1,i+1):
        print(j,end='')
    spaces=2*(n-i)
    for k in range(spaces):
        print(" ",end="")
    for l in range(j,0,-1):
        print(l,end="")
    print()